js/impress-audio.js

```diff
@@ -0,0 +1,77 @@
+/*
+https://github.com/danielsimons1/impress-audio
+Author: Daniel Simons
+Author Email: daniel.simons1@gmail.com
+Version: 1.0.0
+License: Free General Public License (GPL)
+*/
+(function(document, $, undefined) {
+  'use strict';
+
+  var isPlaying, waitTimeout ,
+    audio,video,media,currPlaying,
+    impressObj = impress(),
+    impressGoto = impressObj.goto;
+  var TimerID;
+  var SpacingTime = 300; //ms
```

Comment:
Wouldn't 

    var spacigTimeMs = 300;

be better? You don't need to remember it's in ms